### PR TITLE
[DOCFIX] doc update for some pool related configurations

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: docs/_data/table/en/user-configuration.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ alluxio.user.block.worker.client.threads:
   The number of threads used by a block worker client pool to talk to a remote block worker. It
   determines the maximum number of Thrift connections between an Alluxio client and an Alluxio
   worker.
+alluxio.user.block.worker.client.pool.size.max:
+  The maximum number of block worker client cached in the block worker client pool.
```
Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: `client` -> `clients`

File: docs/_data/table/en/user-configuration.yml [hunk 2]
```diff
@@ -53,6 +57,10 @@ alluxio.user.network.netty.timeout.ms:
   wait for a response from the data server.
 alluxio.user.network.netty.worker.threads:
   How many threads to use for remote block worker client to read from remote block workers.
+alluxio.user.network.netty.channel.pool.size.max:
+  The maximum number of netty channels kept in the netty channel pool.
+alluxio.user.network.netty.channel.pool.gc.threshold.ms:
+  A netty channel is closed if it has been idle for more than the this threshold.
```
Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: `than the this` -> `than this`

File: docs/_data/table/en/user-configuration.yml [hunk 3]
```diff
@@ -53,6 +57,10 @@ alluxio.user.network.netty.timeout.ms:
   wait for a response from the data server.
 alluxio.user.network.netty.worker.threads:
   How many threads to use for remote block worker client to read from remote block workers.
+alluxio.user.network.netty.channel.pool.size.max:
+  The maximum number of netty channels kept in the netty channel pool.
```
Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: Maybe make this language consistent with the block worker one.

File: docs/_data/table/en/user-configuration.yml (hunk 1 above)
Comment (peisun1115) [inline docs/_data/table/en/user-configuration.yml]: done

File: docs/_data/table/en/user-configuration.yml (hunk 3 above)
Comment (peisun1115) [inline docs/_data/table/en/user-configuration.yml]: done

File: docs/_data/table/en/user-configuration.yml (hunk 2 above)
Comment (peisun1115) [inline docs/_data/table/en/user-configuration.yml]: done

**Comment by peisun1115** [comment]
@calvinjia PTAL

**Pull request merged**